core/roboconf-dm/src/main/java/net/roboconf/dm/internal/api/impl/ApplicationMngrImpl.java

```diff
@@ -351,4 +357,25 @@ private ManagedApplication createApplication(
 		this.logger.info( "Application " + name + " was successfully created from the template " + tpl + "." );
 		return ma;
 	}
+
+
+	@Override
+	public void bindApplication( ManagedApplication ma, String applicationTemplateName, String applicationName )
```

Comment:
Is the applicationTemplateName param necessary ?
In the method, we check that it is equal to app.getTemplate().getName() - so if we can obtain it, we don't need it in the param list...